docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+---
+sidebar_label: jira
```

Comment:
Is there an icon we can use to indicate this as an enterprise feature?

Reply:
Good suggestion, though I would just say `Enterprise` and not `Enterprise Only`.

Reply:
We probably need a "beta" label as well.

Reply:
<img width="741" alt="Screenshot 2025-07-02 at 12 55 17 PM" src="https://github.com/user-attachments/assets/ca6e41c1-ef0f-406e-8c9b-e20f2c177d6d" />